applications/asset_tracker_v2/tests/gps_module/src/gps_module_test.c

```diff
@@ -68,6 +68,20 @@ int test_suiteTearDown(int num_failures)
 	return generic_suiteTearDown(num_failures);
 }
 
+void setUp(void)
+{
+	mock_gps_Init();
+	mock_modules_common_Init();
+	mock_event_manager_Init();
+}
+
+void tearDown(void)
+{
+	mock_gps_Verify();
+	mock_modules_common_Verify();
+	mock_event_manager_Verify();
+}
```

Comment:
CMock should be able to do this automatically, IIRC: 

https://github.com/ThrowTheSwitch/Unity/blob/aeed24c78b3c7e0ef7aacf67f17fa585d35594c3/auto/generate_test_runner.rb#L277-L283